src/test/java/CartTests.java

```diff
@@ -0,0 +1,66 @@
+import hw_at_18_rozetka_page_objects.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CartTests extends BaseTest {
+
+    @Test
+    public void verifyEmptyCart() {
+
+        final String EMPTY_CART_TEXT = "Кошик порожній";
+
+        HomePage homePage = new HomePage(getDriver());
+        homePage.clickCartIcon();
+
+        CartPage cartPage = new CartPage(getDriver());
+
+        Assert.assertTrue(cartPage.getPageTitle().contains(EMPTY_CART_TEXT) && cartPage.imageIsPresent());
```

Comment:
This assertion should be splitted to two separate

Reply:
fixed